shell/app/common/components/contractive-filter/index.tsx

```diff
@@ -138,7 +138,7 @@ const OptionItem = (props: IOptionItemProps) => {
     </div>
   );
 };
-
+const firstShowLength = 200;
```

Comment:
make it as a configuration prop, default is 200, if set to null. Then no limit at all